src/install_rules.ml

```diff
@@ -226,33 +226,38 @@ module Gen(P : Params) = struct
     | Default -> true
     | Opam _  -> false
 
+  let process_entries (package : Package.t) entries =
+    let install_paths =
+      Install.Section.Paths.make ~package:package.name ~destdir:Path.root ()
+    in
+    local_install_rules entries ~package:package.name ~install_paths
+
+  (* entries have to be preliminary processed with [process_entries] *)
   let install_file (package : Package.t) entries =
-    let entries =
+    let install_paths =
+      Install.Section.Paths.make ~package:package.name ~destdir:Path.root ()
```

Comment:
It seems that we are effectively calling this 3 times: once here and twice through `process_entries`. It'd be nice if we could call it only once, though it's probably not a big deal in practice sowe can keep the code as it if it's annoying to change.

Reply:
I had thought about that and was not sure what would have been better. I refactored it out, it will be called once at the price of a lookup in the map and a call to `Option.value_exn` (that should be safe in this case as we are computing the install paths for all the packages). It can be extracted further instead of staying in the pre-init step but I think it is more consistent to keep it all contained